jetty-webapp/src/test/java/org/eclipse/jetty/webapp/jsp/FakeJspServlet.java

```diff
@@ -0,0 +1,37 @@
+//
```

Comment:
If this and the other test servlets aren't referenced anywhere else, can we make them inner classes of the ForcedServletTest?

Reply:
done